AltStore/Resources/apps.json

```
@@ -7,14 +7,14 @@
       "name": "AltStore",
       "bundleIdentifier": "com.rileytestut.AltStore",
       "developerName": "Riley Testut",
-      "version": "1.4.8",
-      "versionDate": "2021-10-28T13:00:00-07:00",
-      "versionDescription": "This update supports installing Fugu14-based untethered jailbreaks.\n\nPREVIOUS UPDATE\n\nFEATURES\n• Supports iOS 15.1 \n• Adds \"Open\" context menu action for active apps\n\nIMPROVEMENTS\n• Improves \"Sideloaded App Limit Reached\" error message\n• Asks you to deactivate an app first when installing a new app without any free app slots\n\nBUG FIXES\n• Fixes widget displaying incorrect number of days until expiration\n• Fixes widget layout on iOS 15\n• Fixes \"Refresh All Apps\" shortcut timing out\n\n⚠️ If you have already updated to iOS 15.1, you must manually install this update by updating AltServer to the latest version and using it to re-install AltStore. You do not need to delete AltStore first.",
-      "downloadURL": "https://cdn.altstore.io/file/altstore/apps/altstore/1_4_8.ipa",
+      "version": "1.4.9",
+      "versionDate": "2022-03-04T10:00:00-08:00",
```

```suggestion
      "versionDate": "2022-03-07T09:00:00-08:00",
```

Should be closer to release date, and probs 9am to make sure we don’t accidentally release update before versionDate